deploy/charts/harvester/charts/kubevirt/values.yaml

```diff
@@ -16,231 +16,231 @@ fullnameOverride: ""
+#  ##
+#  # monitorNamespace: "openshift-monitor"
+#
+#  ## Specify the name of the Prometheus service account that needs read-access to KubeVirt endpoints in OpenShift,
+#  ## defaults to "prometheus-k8s".
+#  # monitorAccount: "prometheus-k8s"
+#
+#  ## Specify the uninstall strategy of KubeVirt resource,
+#  ## select from [RemoveWorkloads, BlockUninstallIfWorkloadsExist],
+#  ## defaults to "RemoveWorkloads".
+#  ## -- If specifies to BlockUninstallIfWorkloadsExist, we cannot drop KubeVirt CRD directly, we need to change this field to blank or RemoveWorkloads.
+#  ##
+#  uninstallStrategy: "RemoveWorkloads"
+#
+#  ## Specify the strategy of KubeVrit bundle components certificate rotation.
+#  ##
+#  certificateRotateStrategy:
+#
+#    ## Specify the configuration of self signed certificate.
+#    ##
+#    selfSigned:
+#
+#      ## Specify the interval of CA rotation,
+#      ## defaults to "168h", 7d.
+#      ##
+#      caRotateInterval: "168h"
+#
+#      ## Specify the interval of Certificate rotation,
+#      ## defaults to "24h", 1d.
+#      ##
+#      certRotateInterval: "24h"
+#
+#      ## Specify the interval of CA overlap,
+#      ## defaults to "24h", 1d.
+#      caOverlapInterval: "24h"
+#
+#  ## Specify the default configuration of KubeVirt virtual machine.
+#  ##
+#  configuration:
+#
+#    ## Specify the directory that contains the EFI roms,
+#    ## this argument will be passed to "virt-launcher" as "--ovmf-path",
+#    ## defaults to "/usr/share/OVMF".
+#    ##
+#    ovmfPath: "/usr/share/OVMF"
+#
+#    ## Specify the SELinux type, this argument will fill into `spec.securityContext.selinuxOptions.type` of the generated Pod for "virt-launcher",
+#    ## and also fill into `spec.containers[?(.name!=compute)].securityContext.selinuxOptions.type` and `spec.containers[?(.name!=compute)].securityContext.selinuxOptions.level=s0`.
+#    ##
+#    selinuxLauncherType: ""
+#
+#    ## Specify the version regex of guest agent, which is used for processing the Condition(AgentVersionNotSupported) of VirtualMachineInstance,
+#    ## defaults to "[3.*, 4.*]".
+#    #
+#    supportedGuestAgentVersions:
+#      - "3.*"
+#      - "4.*"
+#
+#    ## Specify the default CPU model of VirtualMachineInstance, selects from https://github.com/libvirt/libvirt/tree/master/src/cpu_map,
+#    ## defaults to "host-model".
+#    ##
+#    cpuModel: "host-model"
+#
+#    ## Specify the default CPU request of VirtualMachineInstance(`spec.domain.requests.cpu`),
+#    ## defaults to "100m"
+#    ##
+#    # cpuRequest: "100m"
+#
+#    ## Specify the machine type regex for validating the emulation of VirtualMachineInstance,
+#    ## defaults to discover automatically, "pseries*" if arch is ppc64le, otherwise "q35*,pc-q35*".
+#    ##
+#    emulatedMachines: []
+#
+#    ## Specify the machine type for filling VirtualMachineInstance(`spec.domain.machine.type`),
+#    ## defaults to discover automatically, "pseries" if arch is ppc64le, otherwise "q35".
```

Comment:
Comment out in [this format](https://github.com/rancher/harvester/blob/cc3244910710325253abf19c8f4f813af3ec90ec/deploy/charts/harvester/charts/kubevirt/values.yaml#L19-L22) for a better look

Reply:
updated, thanks